### PR TITLE
get support page tests

### PR DESCRIPTION
**Pull request by yonasberhe23**
### Summary
This PR covers the functionality within the Get Support page
View Test Plan https://github.com/rancher/qa-tasks/issues/764

### Additional Notes:
- added `navTo` method for the Home page and updated those tests
- enabled skipped tests after [bug fix](https://github.com/rancher/rancher-docs/issues/727)

### CODE REVIEW

File: cypress/e2e/po/pages/get-support.po.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+   * https://docs.cypress.io/guides/references/trade-offs#Multiple-tabs
+   * @param index
+   * @param isNewTab
+   * @returns
+   */
+  clickSupportLink(index: number, isNewTab?: boolean) {
+    if (isNewTab) {
+      this.supportLinks().eq(index).then((el) => {
+        expect(el).to.have.attr('target');
+      }).invoke('removeAttr', 'target')
```
Comment (richard-cox) [inline cypress/e2e/po/pages/get-support.po.ts:47]: Comment only. 

I'm not really a fan of changing the dom, but it's exactly what cypress suggest we do

**Review by richard-cox — CHANGES_REQUESTED** [review]
Only one thing to address, the other comment is just an observation.

File: cypress/e2e/po/pages/get-support.po.ts (hunk 1 above)
Comment (yonasberhe23) [reply cypress/e2e/po/pages/get-support.po.ts:47]: Same here, hopefully they will support multiple tabs soon!

**Review by richard-cox — APPROVED** [review]
LGTM. Please only merge once we have forked for 2.8q1. 

Also need to be careful of https://github.com/rancher/dashboard/pull/9887
- if that merges before please update this pr
- if this merges after please leave a comment on that PR

**Pull request merged**